src/navigation/sectionalTabs/sectionalTabs.jsx

```diff
@@ -0,0 +1,613 @@
+    /**
+     * onChange active tab callback
+     */
+    onChange: PropTypes.func,
+    /**
+     * List of the tabs objects
+     */
+    items: PropTypes.arrayOf(
+        PropTypes.shape({
+            key: PropTypes.string.isRequired,
+            title: PropTypes.string.isRequired,
+            getContent: PropTypes.func,
+            onClick: PropTypes.func,
+        }),
+    ),
+    /**
+     * frequency of onResize recalculation fires
+     */
+    resizeThrottle: PropTypes.number,
+    /**
+     * Use this prop to programatically change the selected tab
+     */
+    selectedTabKey: PropTypes.oneOfType([PropTypes.number, PropTypes.string]),
+    /**
+     * Renders the content set inside the item object under the tabs panel
+     */
+    withContent: PropTypes.bool,
+};
+
+const defaultProps = {
+    beforeChange: undefined,
+    classes: null,
+    items: [],
+    onChange: undefined,
+    resizeThrottle: 100,
+    selectedTabKey: undefined,
+    withContent: false,
+};
+
+const useStyles = (theme) => {
+    const borderColorContrastPrimary = get(theme, 'palette.border.contrastPrimary');
+    const borderColorSecondary = get(theme, 'palette.border.secondary');
+    const colorActivePrimary = get(theme, 'palette.active.primary');
+    const textColorSecondary = get(theme, 'palette.text.secondary');
+    const textColorPrimary = get(theme, 'palette.text.primary');
+    const colorHighlight = get(theme, 'palette.cyan[500]');
+
+    const styles = {
+        root: {
+            position: 'relative',
+            '& .button_dropdown': {
+                margin: '0',
+                width: HIDDEN_TABS_ICON_WIDTH,
+                '& .icon': {
+                    marginRight: '00 !important',
+                },
+                '&-open': {
+                    '& .icon-use-path': {
+                        fill: `${colorHighlight} !important`,
+                    },
+                },
+            },
+        },
+        sectionalTabsPanel: {
+            display: 'flex',
+            flexWrap: 'wrap',
+        },
+        sectionalTabsPanelContent: {
+            padding: '10px 0 10px 0',
+            borderTop: `1px solid ${borderColorSecondary}`,
+        },
+        sectionalTab: {
+            cursor: 'pointer',
+            zIndex: 1,
+            whiteSpace: 'nowrap',
+            padding: '10px 20px 0 0',
+            outline: 'none',
+        },
+        sectionalTabLabel: {
+            fontSize: 14,
+            color: textColorSecondary,
+            paddingBottom: 10,
+            transition: 'color 0.1s, border-bottom 0.1s',
+            borderBottom: `2px solid ${borderColorContrastPrimary}`,
+            '&:hover': {
+                color: textColorPrimary,
+            },
+        },
+        sectionalTabLabelSelected: {
+            color: `${textColorPrimary} !important`,
+            borderBottom: `2px solid ${colorActivePrimary}`,
+        },
+    };
+
+    return styles;
+};
+
+class Tabs extends Component {
```

Comment:
Please change the component name from `Tabs` to `SectionalTabs` and add a component description here.

Reply:
Done